WalletWasabi/CoinJoin/Coordinator/Coordinator.cs

```diff
@@ -322,6 +326,43 @@ private async void Round_StatusChangedAsync(object sender, CoordinatorRoundStatu
 								}
 							}
 						}
+
+						var newIntervalTriggered = Stats.Register(round.CoinJoin);
+						// Adjust timeout: https://github.com/zkSNACKs/WalletWasabi/issues/2940
+						if (newIntervalTriggered && Stats.GetAverageWcqs().Count() > 2)
```

Comment:
If I restart the server, the timeout won't be modified for 2 days. Is this right? What if I restart it in an edge case.